string Class
float64 probability
int64 tracking_index
int64 xmin
int64 ymin
int64 xmax
int64 ymax

